scipy/optimize/_qap.py

```diff
@@ -0,0 +1,789 @@
+import numpy as np
+import operator
+from . import (linear_sum_assignment, minimize_scalar, OptimizeResult)
```

Comment:
`minimize_scalar` is not needed anymore. It might be an idea to run pyflakes on all the new code.

Reply:
Removed